### ISSUE TITLE
Migrate to depend MPAndroidChart to MPAndroidChart-Realm

### ISSUE THREAD

**Issue by bkerensa** [Glucosio/glucosio-android#280]
PhilJay now has a Realm MPAndroidChart Specific Library we might want to land this next release 
https://github.com/PhilJay/MPAndroidChart-Realm/

**Comment by SubhrajyotiSen** [comment]
Can I try working on this?

**Comment by bkerensa** [comment]
Yes

**Comment by ghost** [comment]
@SubhrajyotiSen still interested in working on this?

**Comment by pt2121** [comment]
can i take a stab at this?

**Comment by paolorotolo** [comment]
Sure! Feel free to assign this issue to yourself.

**Comment by pt2121** [comment]
i might be wrong but it seems you transform data before displaying it, e.g. with `glucoseToMmolL`, `convertDate`, etc. so, i don't think we can take much advantage of _MPAndroidChart-Realm_.

am i missing something?

**Comment by emartynov** [comment]
Yes, you're right that we do some small data migrations depends on user preference.
